Revit_Core_Engine/Query/ParameterGroupTypeId.cs

```diff
@@ -0,0 +1,112 @@
+/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2025, the respective contributors. All rights reserved.
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using Autodesk.Revit.DB;
+using System.ComponentModel;
+using BH.oM.Base.Attributes;
+using System.Collections.Generic;
+using System.Reflection;
+using System;
+using System.Linq;
+
+namespace BH.Revit.Engine.Core
+{
+    public static partial class Query
+    {
+#if REVIT2020 || REVIT2021 || REVIT2022 || REVIT2023 || REVIT2024
+        /***************************************************/
+        /****              Public methods               ****/
+        /***************************************************/
+
+        public static BuiltInParameterGroup ParameterGroupTypeId(this string groupName)
```

Comment:
Looking at similar cases of `Spec` and `Unit`, I would argue there should be 3 methods:
- `GetGroupTypeId(Definition)` returning the enum or `ForgeTypeId`
- `GroupName` returning name of  the property of `GroupTypeId` static class that is correspondent to the provided value
- `GroupFromName` - inverse of the above

...please see similar methods for `Unit` for reference. If we need label on top of the above, let's add another method (the one you added seems to be OK), but any string-based matching should happen around the property names of the static class that lists out `ForgeTypeIds` of given category.